packages/keystatic/src/form/fields/document/DocumentEditor/render-element.tsx

```diff
@@ -143,11 +133,7 @@ export const renderElement = (props: RenderElementProps) => {
     default:
       let { textAlign } = props.element;
       return (
-        <p
-          className={blockElementSpacing}
-          style={{ textAlign }}
-          {...props.attributes}
-        >
+        <p style={{ textAlign }} {...props.attributes}>
```

Comment:
We should get rid of text-alignment as a feature.

Reply:
This is now off by default as of #538

You can still turn it on though, and it adds a Markdoc attribute to the block

This seems fine, because aligning text is still a valid feature for content editing, but it's now an explicit opt-in (otherwise people would have to use custom component blocks to make it possible and that would be generally worse in all ways)